docs/index.md

```diff
@@ -13,13 +13,100 @@ myst:
 
 This is the community-maintained documentation for the Plone content management system.
 
-This documentation is for:
+:::::{grid} 1 1 2 2
+:gutter: 3 3 4 5
 
-- Editors who write and update content on a site
-- Site administrators who install Plone and add-ons, and set up a site
-- Designers who create site themes
-- Deployers who configure servers for site hosting
-- Developers who customize a site’s capabilities, create add-ons, and contribute to Plone itself
+::::{grid-item-card} 
+:link: install/index
+:link-type: doc
+{octicon}`zap;1.5em;sd-text-info`
+Get started
+^^^
+This part of the documentation helps you find the best way to get started with Plone, depending on what you want to do.
++++
+```{button-ref} {doc} install/index
+:color: primary
+:expand:
+Go to Get started
+``` 
+::::
+
+::::{grid-item-card} 
+:link: user-guide/index
+:link-type: doc
+{octicon}`person;1.5em;sd-text-info`
+User guide 
+^^^
+This part of the documentation helps you understand how to use Plone effectively, covering essential features, navigation, content management, and common tasks for users and administrators.
++++
+```{button-ref} {doc} user-guide/index
+:color: primary
+:expand:
+Go to User guide
+```
+::::
+
+::::{grid-item-card}
+:link: developer-guide/index
+:link-type: doc
+{octicon}`person-add;1.5em;sd-text-info`
+Developer guide
+^^^
+This part of the documentation describes how to develop a Plone site.
++++
+```{button-ref} {doc} developer-guide/index 
+:color: primary
+:expand:
+Go to Developer guide
+```
+::::
+
+::::{grid-item-card} 
+:link: reference-guide/index
+:link-type: doc
+{octicon}`bookmark;1.5em;sd-text-info`
+Reference guide 
+^^^
+This part of the documentation provides explanation of APIs, functions, modules, and objects included in Plone, describing what they are and what they do.
++++
+```{button-ref} {doc} reference-guide/index 
+:color: primary
+:expand:
+Go to Reference guide
+```
+::::
+
+::::{grid-item-card} 
+:link: conceptual-guides/index
+:link-type: doc
+{octicon}`light-bulb;1.5em;sd-text-info`
+Conceptual guides 
+^^^
+This part of the documentation provides explanation of concepts to deepen and broaden your understanding of Plone.
++++
+```{button-ref} {doc} conceptual-guides/index 
+:color: primary
+:expand:
+Go to Conceptual guides
+```
+::::
+
+::::{grid-item-card} 
+:link: contributing/index
+:link-type: doc
+{octicon}`book;1.5em;sd-text-info`
```

Comment:
The book icon represents documentation, whereas the pull request icon relates better to making a contribution.

```suggestion
{octicon}`git-pull-request;1.5em;sd-text-info`
```